### ISSUE TITLE
Loading data for RandomiseGroupLevel

### ISSUE THREAD

**Issue by cmehta126** [wanderine/BROCCOLI#35]
I am performing permutation tests on results and want to use RandomiseGroupLevel. To test it out, I placed placed spatial volumes (beta-maps) for 10 subjects into a single nifti file and then created files for design/contrast matrices for 10 subjects. I ran the following to test the program.
> RandomiseGroupLevel a.nii -design design.mat -contrasts design.con

The output was:
> Warning: No mask being used, doing permutations for all voxels.
> Input data is a single volume, nothing to permute!

I am not using a mask in this test. But, there are definitely 10 volumes in  the "a.nii" file. Does this commonly occur? 

I created the "a.nii" file by using AFNI's 3dbucket to compile them and then 3dAFNItoNIFTI to convert to nifti format. Could this be a source of error?

Thank you.

**Comment by wanderine** [comment]
Can you run

3dinfo a.nii

and send me the output

Den 28 mars 2017 19:37 skrev "cmehta126" <notifications@github.com>:

**Comment by cmehta126** [comment]
Thank you for the quick reply. I think I just figured out a solution. Instead of using 3dbucket for combining beta-maps for subjects, I combined them with "3dTcat" and the Broccoli program is able to read the data. The "3dTcat" represents the combined volumes as a 3d-time series, whereas "3dbucket" does not represent them as such in the header. It is likely the Nifti reader in Broccoli does not like that.
